src/qt/veil/addresseswidget.cpp

```diff
@@ -256,24 +256,33 @@ void AddressesWidget::handleAddressClicked(const QModelIndex &index){
 
     listView->setCurrentIndex(updatedIndex);
     QRect rect = listView->visualRect(index);
-    QPoint pos = rect.topRight();
-    pos.setX(pos.x() - (DECORATION_SIZE * 2));
-    pos.setY(pos.y() + (DECORATION_SIZE));
+
     const QString constType = type;
-    if(!this->menu) this->menu = new AddressesMenu(constType , updatedIndex, this, this->mainWindow, this->model);
+    if(!this->menu) this->menu = new AddressesMenu(constType , updatedIndex, mainWindow->getGUI(), this->mainWindow, this->model);
```

Comment:
```suggestion
    if (!this->menu)
        this->menu = new AddressesMenu(constType, updatedIndex, mainWindow->getGUI(), this->mainWindow, this->model);
```

Reply:
I will likely not be getting to this PR anytime soon but commenting as noted.